cg/services/post_processing/pacbio/run_file_manager/run_file_manager.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+from cg.constants import FileExtensions
+from cg.constants.pacbio import PacBioDirsAndFiles
+from cg.services.post_processing.abstract_classes import RunFileManager
+from cg.services.post_processing.exc import PostProcessingFileNotFoundError
+from cg.services.post_processing.pacbio.run_data_generator.run_data import PacBioRunData
+from cg.services.post_processing.validators import validate_files_exist
+from cg.utils.files import get_files_matching_pattern
+
+
+class PacBioRunFileManager(RunFileManager):
+
+    def get_files_to_parse(self, run_data: PacBioRunData) -> list[Path]:
+        """Get the file paths required by the PacBioMetricsParser."""
+        run_path: Path = run_data.full_path
+        files_to_parse: list[Path] = [self._find_ccs_report_file(run_path)]
+        files_to_parse.extend(self._get_unzipped_report_files(run_path))
+        return files_to_parse
+
+    def get_files_to_store(self, run_data: PacBioRunData) -> list[Path]:
+        """Get the files to store for the PostProcessingHKService."""
+        run_path: Path = run_data.full_path
+        files_to_store: list[Path] = self._find_hifi_files(run_path)
+        files_to_store.extend(self.get_files_to_parse(run_data))
+        return files_to_store
+
+    @staticmethod
+    def _find_ccs_report_file(run_path: Path) -> Path:
+        """Return the path to the the CCS report file."""
+        statistics_dir: Path = Path(run_path, PacBioDirsAndFiles.STATISTICS_DIR)
+        files: list[Path] = get_files_matching_pattern(
+            directory=statistics_dir, pattern=f"*{PacBioDirsAndFiles.CCS_REPORT_SUFFIX}"
+        )
+        if not files:
+            raise PostProcessingFileNotFoundError(f"No CCS report file found in {statistics_dir}")
+        return files[0]
+
+    @staticmethod
+    def _get_unzipped_report_files(run_path: Path) -> list[Path]:
```

Comment:
```suggestion
    def _get_report_files(run_path: Path) -> list[Path]:
```
I don't think we need to care whether they have been zipped or not previosuly

Reply:
fixed